### ISSUE TITLE
Accessibility - PDF Text is Blurry

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Child-Labor-Android#67]
When viewing some of the PDFs on the Android application, the text is blurry and even harder to read because it is small.

Combining issues #5 and #14 before closing them.

We need to look at a better way to present that content so it's easier to read for users and that might be to not have PDFs as part of the application.

More research is needed. I hope that Android finds a better way to include PDFs in their native applications.

**Comment by jfc3-dol** [comment]
Still having issues with PDFs having blurry text/content/

**Comment by jfc3-dol** [comment]
Not able to look at any PDFs in the application, so I'm going to close this issue and open a new one.

**Comment by jfc3-dol** [comment]
Reopened issue because the PDFs for the individual countries are blurry and hard to read still.

**Comment by clguidi** [comment]
John, unfortunately this is not something the app team can address.

**Comment by jfc3-dol** [comment]
@clguidi,

Yes, we are keeping it open so we can check each year if the operating system or software letting people view PDFs improves because it makes it difficult for those with poor vision to read the text.
